python-sdk/tests/benchmark/analyse.py

```diff
@@ -131,24 +167,55 @@ def analyse_results(results_filepath):
 
     summary = mean_by_dag[SUMMARY_FIELDS]
 
+    content = ""
     # print Markdown tables per database
     for database_name in summary["database"].unique().tolist():
-        print(f"\n### Database: {database_name}\n")
-        print(summary[summary["database"] == database_name].to_markdown(index=False))
+        content = content + f"\n\n### Database: {database_name}\n"
+        content = content + str(
+            summary[summary["database"] == database_name].to_markdown(index=False)
+        )
+
+    if output_filepath:
+        f = open(output_filepath, "w")
+        f.write(content)
+    else:
+        print(content)
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Trigger benchmark DAG")
-    parser.add_argument(
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument(
         "--results-filepath",
         "-r",
         type=str,
         help="NDJSON local path (/path/to/file.ndjson) or Google "
         "cloud storage path (gs://bucket/sample.ndjson) containing the results for a benchmark run",
     )
+    group.add_argument(
+        "--git-sha",
```

Comment:
@utkarsharma2 It may be worth giving an argument name which makes clear this relates to BQ. Something like: `--bq-git-sha` and `-b` WDYT?

Every result row has a git hash, so a generic argument `--git-sha` could make sense for analysing JSON files as well.

Reply:
@tatiana Make sense.